corehq/apps/reports/standard/monitoring.py

```diff
@@ -649,12 +649,14 @@ class FormCompletionTimeReport(WorkerMonitoringFormReportTableBase, DatespanMixi
 
     @property
     @memoized
-    def selected_xmlns(self):
-        return SingleFormByApplicationFilter.get_value(self.request, self.domain)
+    def selected_form_data(self):
+        data = FormsByApplicationFilter.get_value(self.request, self.domain)
+        if len(data) == 1 and data.values()[0]['xmlns']:
+            return data.values()[0]
 
     @property
     def headers(self):
-        if self.selected_xmlns['xmlns'] is None:
+        if self.selected_form_data:
```

Comment:
this seems like the opposite condition? shouldn't it be `if not self.selected_form_data`